docs/setup/compose_based.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Composed Based Deployment
```

Comment:
It is "compose" based, not "composed" based.

Reply:
Okay